### PR TITLE
Publish the new workflow package

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-4450 .

#### Changes proposed in this Pull Request:

- Updated the version number in package.json
- Updated the ChangeLog
- Removed shared-auth folder.

#### To Test:
- yarn start:example

#### Any specific feedback you are looking for?

- Open this PR once the opened 2 PRs got merged

### CODE REVIEW

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -5,21 +5,17 @@ All notable changes to this project will be documented in this file.
...
--   A `ViewEulaSubscreen` component.
--   A `ContactSupportSubscreen` component.
--   A `BrandedCardContainer` component.
--   Confirm password field error state message in `ChangePasswordForm`.
--   Option to hide the Create Password screen during registration
--   A `ExistingAccountRegisterSuccessSubscreen` component.
--   A `RegistrationWorkflowContext` context, a `RegistrationWorkflowContextProvider` component and a `useRegistrationWorkflowContext` hook.
+-   Reusable screen components.
+-   Support for custom react auth workflow injection.
+-   Support for router customizability.
```
Comment (ektaghag-eaton) [inline login-workflow/CHANGELOG.md:15]: Please mentioned migration guide link here https://github.com/etn-ccis/blui-react-workflows/blob/dev/login-workflow/README.md#migrating-from-v4--v5

File: login-workflow/CHANGELOG.md [hunk 2]
```diff
@@ -5,21 +5,17 @@ All notable changes to this project will be documented in this file.
...
 
 ### Changed
 
--   Changed the verification code styles for the self registration workflow.
+-   Update support for the latest versions of i18n and react-i18n.
```
Comment (ektaghag-eaton) [inline login-workflow/CHANGELOG.md:18]:
``` suggestion
-   Update i18next and react-i18next to latest.
```

Comment (surajeaton) [inline login-workflow/CHANGELOG.md:18]:
```suggestion
-  Updated i18next and react-i18next packages to the latest.
```

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**